card_house/db/client_db.py

```diff
@@ -0,0 +1,75 @@
+import sqlite3
+from card_house.db.cards_db_columns import CardsDBColumns
+
+from typing import List, Any, Dict
+
+from card_house.infrastructure.entities.card_type import CardType
+from card_house.infrastructure.entities.db_card import DBCard
+from card_house.db.iclient_db import IClientDB
+
+
+class CardsDB(IClientDB):
+    def __init__(self, db_name: str):
+        self.db_name = db_name
+        self.create_cards_table()
```

Comment:
Wait what? 
We create a table every time we run the code?!
Oh never mind... I just read the query damn it was quite a rollercoaster Jesus.
Consider moving it out of the const' (maybe to __start__ or something like that).